### PR TITLE
card component

### PR DESCRIPTION
**Pull request by devAmlan**
This is the card component. Please review my PR.

### CODE REVIEW

File: library/component/card/card.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
+.card{
+    width:23rem;
+    min-height:30rem;
+    text-align:left;
+    border:2px solid var(--primary-color);
```
Comment (varun13169) [inline library/component/card/card.css:5]: For better readability.

property:**(space)**value

File: library/component/css/component.css [hunk 2]
```diff
@@ -20,6 +20,9 @@
     --danger-bgcolor:#fa6a676e; 
     --warning-color:#FFAB4C;
     --warning-bgcolor:#ffe2795b; 
+    --shadow-color1:rgba(0, 0, 0, 0.2);
+    --shadow-color2:rgba(0, 0, 0, 0.19);
```
Comment (varun13169) [inline library/component/css/component.css:24]: Use variables.

**Review by varun13169 — CHANGES_REQUESTED** [review]

**Review by Omkar-Ghate — APPROVED** [review]
Use space between property and value. Rest everything looks good.

File: library/component/card/card.css (hunk 1 above)
Comment (devAmlan) [reply library/component/card/card.css:5]: I will update it , thanks Varun

**Comment by devAmlan** [comment]
I will use it , thanks omkar.

File: library/component/css/component.css (hunk 2 above)
Comment (devAmlan) [reply library/component/css/component.css:24]: u mean to change the RGBA into hex ?

**Pull request merged**